Project.toml

```diff
@@ -6,17 +6,21 @@ version = "1.5.0-DEV"
 [deps]
 AbstractAlgebra = "c3fe647b-3220-5bb0-a1ea-a7954cac585d"
 AlgebraicSolving = "66b61cbe-0446-4d5d-9090-1ff510639f9d"
+CodecZlib = "944b1d66-785c-5afd-91f1-9de20f533193"
```

Comment:
This change (and all others in this file) are not needed for the added feature, and should such be dropped from the PR.